aten/src/ATen/native/mps/UnaryConstants.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+struct ErfinvConstant {
+  float a[4] = { 0.886226899, -1.645349621, 0.914624893, -0.140543331 };
+  float b[4] = { -2.118377725, 1.442710462, -0.329097515, 0.012229801 };
+  float c[4] = { -1.970840454, -1.624906493, 3.429567803, 1.641345311 };
+  float d[2] = { 3.543889200, 1.637067800 };
+};
```

Comment:
Thank you for the suggestion. We can make use of constexpr in the objective-c part of the code but we cannot use constexpr in the metal kernel code since the metal kernel code is currently compiled and cached at run time.

Reply:
Yes you are right, the metal kernel is compiled during runtime. I meant, constexpr is a defined keyword in MSL specs, which can be used in metal kernel. 

With all constants here defined in the metal kernel using constexpr, after the kernel is compiled and cached, the constants become literals and not dynamic anymore. We also no longer need to encode and pass the constants to the kernel.

Reply:
ah yea after thinking about it I can simply declare them as:

```
constant float a[4] = {{0.886226899, -1.645349621, 0.914624893, -0.140543331}};
constant float b[4] = {{-2.118377725, 1.442710462, -0.329097515, 0.012229801}};
constant float c[4] = {{-1.970840454, -1.624906493, 3.429567803, 1.641345311}};
constant float d[2] = {{3.543889200, 1.637067800}};
```
directly in the kernel and I wouldn't need any structure as I am not passing any variable data between host and device.